source/_components/light.yeelight.markdown

```diff
@@ -144,3 +179,29 @@ light:
       192.168.1.13:
         name: Front Door
 ```
+
+### {% linkable_title Custom effects %}
+
+This example shows how you can add your custom effects in your configuration.
+
+Possible transitions are `RGBTransition`, `HSVTransition`, `TemperatureTransition`, `SleepTransition`
+
+
+More info about values, transitions in [library docs](https://yeelight.readthedocs.io/en/stable/flow.html)
```

Comment:
More info about transitions and their expected parameters can be found in [python-yeelight documentation].